pkgs/development/python-modules/osqp/default.nix

```diff
@@ -12,11 +12,14 @@
 
 buildPythonPackage rec {
   pname = "osqp";
-  version = "0.6.1";
+  version = "unstable-2020-03-18";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "130frig5bznfacqp9jwbshmbqd2xw3ixdspsbkrwsvkdaab7kca7";
+  src = fetchgit {
+    url = "https://github.com/GuillaumeDesforges/osqp-python";
```

Comment:
Use fetchFromGitHub.
Also include a note of what to revert when OSQP PR XX is closed